frontend/public/components/instantiate-template.tsx

```diff
@@ -251,7 +251,22 @@ class TemplateForm_ extends React.Component<
     }
 
     const template: TemplateKind = obj.data;
-    const parameters = template.parameters || [];
+
+    let parameters = template.parameters || [];
+    if (template?.metadata?.labels['workload.template.kubevirt.io/saphana'] === 'true') {
+      const sapHanaParams = [...parameters];
+      sapHanaParams?.map(
+        (param) =>
+          (param.displayName = param.name
+            .split('_')
+            .map((word) => {
+              const newWord = word[0] + word.substr(1).toLowerCase();
```

Comment:
Unfortunately, I don't think it's going to be as easy as this. If you look at your mockup video, CPU is cased as Cpu. I don't know if there's a utility somewhere that we can use for putting strings in sentence case that accounts for acronyms, but if there is, we should probably consider using it. If we're going to use this for other templates without knowing what parameters they have, it will be impossible to account for all of the possible acronyms and/or other casing issues this may cause.

Reply:
see comment https://github.com/openshift/console/pull/10617#discussion_r763257828

Reply:
removing this block according to comments